services/streammanager/kafka/kafkamanager.go

```diff
@@ -244,10 +252,7 @@ func Produce(jsonData json.RawMessage, producer interface{}, destConfig interfac
 	var errorMessage string
 	partition, offset, err := kafkaProducer.SendMessage(message)
 	if err != nil {
-		returnMessage = fmt.Sprintf("%s error occured.", err.Error())
-		statusCode = GetStatusCodeFromError(err) //400
-		errorMessage = err.Error()
-		pkgLogger.Error(returnMessage)
+		statusCode, returnMessage, errorMessage = makeErrorResponse(err)
```

Comment:
instead of catching them here, lets directly return makeErrorResponse(err) and remove the else, so that the successpath is indented to left.

Reply:
done